lib/sqlite.js

```diff
@@ -1,5 +1,19 @@
 'use strict';
+const {
+  SymbolDispose,
+} = primordials;
 const { emitExperimentalWarning } = require('internal/util');
+const binding = internalBinding('sqlite');
 
 emitExperimentalWarning('SQLite');
-module.exports = internalBinding('sqlite');
+
+// TODO(cjihrig): Move this to C++ once Symbol.dispose reaches Stage 4.
```

Comment:
For educational purposes: why can't we implement this in C++ in the first place?

Reply:
V8 offers helper functions for well known symbols (`GetToStringTag()`, `GetToPrimitive()`, etc.). There doesn't seem to be one for `Symbol.dispose` yet. We could work around it, but I don't want to write more V8 C++ code than I need to for something that could technically still change. If anyone else wants to write this in C++, feel free.

Reply:
@cjihrig so that is the case (the helper functions not existing) even though the symbols are already defined in v8?